practices/slsa.yaml

```diff
@@ -204,6 +204,117 @@ taskDefinitions:
   ###########
   ## BUILD ##
   ###########
+  
+  # Requirements on build process:
+  
+  buildScripted:
+    title: "Build: Scripted build"
+    description: |
+      All build steps were fully defined in some sort of “build script”. The only
+      manual command, if any, was to invoke the build script. Examples:  Build
+      script is Makefile, invoked via make all. Build script is
+      .github/workflows/build.yaml, invoked by GitHub Actions.
+    questions:
+      - text: Are all build steps fully defined in a build script?
+    level: 1
+
+  buildService:
+    title: "Build: Build service"
+    description: |
+      All build steps ran using some build service, not on a developer’s
+      workstation. Examples: GitHub Actions, Google Cloud Build, Travis CI.
+    questions: 
+      - text: Do all build steps run in a build service?
+    level: 2
+
+  buildCode:
+  - title: "Build: Build as code"
+    description: |
+      The build definition and configuration executed by the build service is
+      verifiably derived from text file definitions stored in a version control
+      system. Verifiably derived can mean either fetched directly through a
+      trusted channel, or that the derived definition has some trustworthy
+      provenance chain linking back to version control. Examples: 
```

Comment:
We can do markdown in these descriptions to preserve formatting. For example this one can be:
```markdown
...version control.

Examples:

 - .github/workflows/build.yaml stored...
 - Tekton bundles ...
```

Reply:
Agree